backend/util/util.go

```diff
@@ -176,17 +207,15 @@ func WriteCache(username, filename string, data interface{}) {
 // IsValidQueryType determines if an incoming query is
 // implemented by the service
 func IsValidQueryType(query string) (string, error) {
-	// validTypes := []string{"accountinfo", "pullrequestcommits",
-	// 	"pullrequests", "issuescreated",
-	// 	"prcontributions", "repocontribs"}
 
 	validTypes := map[string]bool{
-		"accountinfo":        true,
-		"pullrequestcommits": true,
-		"pullrequests":       true,
-		"issuescreated":      true,
-		"prcontributions":    true,
-		"repocontribs":       true,
+		"accountinfo":          true,
+		"pullrequests":         true,
+		"involvedissues":       true,
+		"openvsclosedissues":   true,
+		"reposcontributedto":   true,
+		"mergedvsnonmergedprs": true,
+		"podinformation":       true,
```

Comment:
This isn't camel case because the url is allowed to be camel case or just lower case or whatever the user wants. The user's query from the URL is changed to all lower case so the key has to then be lowercase aswell.

So in general the user can use camel case or all caps and it will still match the query

https://github.com/MLH-Fellowship/FellowshipWrapup/blob/2f60ca4727c8a2fc63538984690e6f198f00a488/backend/util/util.go#L220